node/network/bitfield-distribution/src/lib.rs

```diff
@@ -350,7 +356,11 @@ where
 			}
 		})
 		.collect::<Vec<PeerId>>();
-	let interested_peers = util::choose_random_sqrt_subset(interested_peers, MIN_GOSSIP_PEERS);
+	let interested_peers = util::choose_random_subset(
```

Comment:
Why do we only use a subset?

Reply:
I expect that we'll use `MIN_GOSSIP_PEERS` in two cases:
1. Small network (up to 625 validators).
2. The node has just restarted and it's DHT is not populated. It would go higher on the next session, so there is no much point in making this logic more complex than it needs to be.